polyfill/lib/time.mjs

```diff
@@ -168,8 +168,10 @@ export class Time {
     if (!ES.IsTemporalTime(this)) throw new TypeError('invalid receiver');
     if (!ES.IsTemporalTime(other)) throw new TypeError('invalid Time object');
     const largestUnit = ES.ToLargestTemporalUnit(options, 'hours');
-    const [earlier, later] = [this, other].sort(Time.compare);
-    let { hours, minutes, seconds, milliseconds, microseconds, nanoseconds } = ES.DifferenceTime(earlier, later);
+    const comparison = Time.compare(this, other);
+    if (comparison < 0) throw new RangeError('other instance cannot be larger than `this`');
+    let { hours, minutes, seconds, milliseconds, microseconds, nanoseconds } = ES.DifferenceTime(other, this);
+    /*
```

Comment:
I'd just remove the code instead of commenting it out. If we decide to go back to the previous behaviour, we can find it in the git history. (Ditto in several other places below.)

Reply:
Good idea. Done. Resolving.